cmd/wardenkms/Dockerfile

```
@@ -1,16 +1,30 @@
     rm -rf /var/lib/apt/lists/* && \
     useradd -M -u 1000 -U -s /bin/sh -d /data warden && \
     install -o 1000 -g 1000 -d /data
-COPY ./wardenkms /usr/bin/wardenkms
 
 ARG WASMVM_VERSION
-ARG ARCH
-ARG WASMVM_CHECKSUM
-ADD --checksum=sha256:${WASMVM_CHECKSUM} --chmod=444 --chown=nobody:nogroup https://github.com/CosmWasm/wasmvm/releases/download/${WASMVM_VERSION}/libwasmvm.${ARCH}.so /lib/libwasmvm.${ARCH}.so
+ARG WASMVM_AMD64_CHECKSUM
+ARG WASMVM_ARM64_CHECKSUM
+ADD --checksum=sha256:${WASMVM_AMD64_CHECKSUM} --chmod=755 https://github.com/CosmWasm/wasmvm/releases/download/${WASMVM_VERSION}/libwasmvm.x86_64.so /usr/lib/libwasmvm.x86_64.so
```

Hadolint (2.12.0)

[error] invalid flag: --checksum

(DL1000)